Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs

```diff
@@ -215,6 +234,32 @@ private void HandleTestCompletion(object sender, TestCompletedEventArgs e)
             Tests.Refresh();
         }
 
+        public IRewritingManager RewritingManager { get; }
+
+        public string TestToggleLabel
```

Comment:
This property appears to be unused?

Reply:
Should be a private field, unless it's accessed from outside the class... or removed altogether if it's just unused.